models/Element/AbstractElement.php

```diff
@@ -527,7 +529,7 @@ public function getVersions()
     public function __sleep()
     {
         $parentVars = parent::__sleep();
-        $blockedVars = ['dependencies', 'parent'];
+        $blockedVars = ['dependencies'];
```

Comment:
why was that removed again?
I guess that is also the reason for the failing tests....

Reply:
Yes, I was working on the other `__sleep` and `__wakeup` method refactoring and I might have mixed up those two. Sorry, fixed it.